### PR TITLE
Fix typings

### LINKED ISSUE #2: Typescript declaration file
Thanks for this great plugin! 

Unfortunately I am getting this error when trying to use the plugin in a typescript project. Can you please help

TS7016: Could not find a declaration file for module 'vue3-apexcharts'. 'C:/Trooptravel/Development/troop-explore-report/node_modules/vue3-apexcharts/dist/vue3-apexcharts.common.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/vue3-apexcharts` if it exists or add a new declaration (.d.ts) file containing `declare module 'vue3-apexcharts';`

### LINKED ISSUE #18: Problem with typescript
```
node_modules/vue3-apexcharts/typings/vue-apexcharts.d.ts:2:30 - error TS2307: Cannot find module 'vue/types/plugin' or its corresponding type declarations.

2 import { PluginObject } from 'vue/types/plugin';
                               ~~~~~~~~~~~~~~~~~~

node_modules/vue3-apexcharts/typings/vue-apexcharts.d.ts:5:42 - error TS2709: Cannot use namespace 'Vue' as a type.

5 interface VueApexChartsComponent extends Vue {
                                           ~~~

node_modules/vue3-apexcharts/typings/vue-apexcharts.d.ts:36:59 - error TS2709: Cannot use namespace 'Vue' as a type.

36 declare const VueApexCharts: Component & ComponentOptions<Vue> & PluginObject<any>;
            
```

Thanks for the great project! Unfortunately I've been unable to use it due to this error. Any recommendations?

### PR DESCRIPTION
**Pull request by IlCallo**
Closes #2
Closes #18

See #18 for context

`setLocale` types are missing as haven't found how it should work anywhere

I tested it locally on my project using it and seems it works. I run the demo examples and they're working too.
If you have a way to test it further on your own, please do

### CODE REVIEW

File: typings/vue-apexcharts.d.ts [hunk 1]
```diff
@@ -24,6 +23,7 @@ interface VueApexChartsComponent extends Vue {
   zoomX(min: number, max: number): void;
   toggleDataPointSelection(seriesIndex: number, dataPointIndex?: number): any;
   appendData(newData: any): Promise<void>;
+  addText(text: string): void;
```
Comment (junedchhipa) [inline typings/vue-apexcharts.d.ts:26]: addText method is no longer available. Please remote it.

File: src/vue3-apexcharts.js [hunk 2]
```diff
@@ -262,13 +268,34 @@ const vueApexcharts = defineComponent({
...
+      updateOptions,
+      updateSeries,
+      toggleSeries,
+      showSeries,
+      hideSeries,
+      resetSeries,
+      zoomX,
+      toggleDataPointSelection,
+      appendData,
+      addText,
```
Comment (junedchhipa) [inline src/vue3-apexcharts.js:285]: addText method is no longer available. Please remove it

File: src/vue3-apexcharts.js [hunk 3]
```diff
@@ -182,6 +183,10 @@ const vueApexcharts = defineComponent({
       return chart.value.appendData(newData);
     };
 
+    const zoomX = (start, end) => {
+      return chart.value.zoomX(start, end);
+    };
+
     const addText = options => {
```
Comment (junedchhipa) [inline src/vue3-apexcharts.js:190]: addText method is no longer available. Please remove it

**Review by junedchhipa — CHANGES_REQUESTED** [review]
Thanks for the PR.
I have added 3 comments to remove the `addText` method

**Comment by IlCallo** [comment]
Done

**Review by junedchhipa — APPROVED** [review]

**Comment by GTB3NW** [comment]
TYSM for these changes. I'll give it a go and see if everything is working for me :)

**Comment by GTB3NW** [comment]
@junedchhipa would it be possible to get this tagged please? Installing via commit is not working (probably me doing something wrong).

**Comment by junedchhipa** [comment]
I will release a new version very soon (in the next 24-48 hours)

**Comment by GTB3NW** [comment]
> I will release a new version very soon (in the next 24-48 hours)

Smashing, glad to hear it! :)

**Pull request merged**